[DOC] Add language for early-access (#60776)
saltstack/salt: SECURITY.md

-----END PGP PUBLIC KEY BLOCK-----
```

## Reporting a Vulnerability - Private Disclosure Process

> The Salt Project Security Team is available at security@saltstack.com for

Reviewer: need to update the email address

> security-related bug reports or questions. Emails will be addressed within 3 business days.

Reviewer 2: ```suggestion
security-related bug reports or questions. Emails will be addressed as soon as possible.
```

-----END PGP PUBLIC KEY BLOCK-----
```

## Reporting a Vulnerability - Private Disclosure Process

The Salt Project Security Team is available at security@saltstack.com for
security-related bug reports or questions. Emails will be addressed within 3 business days.

We request the disclosure of any security-related bugs or issues be reported
non-publicly until such time as the issue can be resolved and a security-fix
release can be prepared. At that time we will release the fix and make a public
announcement with upgrade instructions and download locations.

**IMPORTANT: Do not file public issues on GitHub for security vulnerabilities**

## Proposed Email Content

Provide a descriptive subject line and in the body of the email include the following information:

* Basic identity information, such as your name and your affiliation or company.

> * Detailed steps to reproduce the vulnerability  (POC scripts, screenshots, and logs are all helpful to us).

Reviewer 3: Two spaces before the `(`

* Description of the effects of the vulnerability on Salt and the related hardware and software configurations, so that the VMware Security Team can reproduce it.
* How the vulnerability affects Salt usage and an estimation of the attack surface, if there is one.
* List other projects or dependencies that were used in conjunction with Salt to produce the vulnerability.

## When to report a vulnerability

* When you think Salt has a potential security vulnerability.
* When you suspect a potential vulnerability but you are unsure that it impacts Salt.
* When you know of or suspect a potential vulnerability on another project that is used by Salt.

## Security response procedure

We take security and the trust of our customers and users very
seriously. Our disclosure policy is intended to resolve security issues as
quickly and safely as is possible.

1.  A security report sent to security@saltstack.com is assigned to a team
    member. This person is the primary contact for questions and will

>     coordinate the fix, release, and announcement. 

Reviewer 3: Trailing space

>     coordinate the fix, release, and announcement. 

Reviewer 4: ```suggestion
    coordinate the fix, release, and announcement.
```

    actively supported. Back-ports of the fix are made to any old releases that
    are actively supported.

4.  A new release is created and pushed to all affected repositories. The
    release documentation provides a full description of the issue, plus any
    upgrade instructions or other relevant details.

5.  An announcement is made to the
    [salt-users](https://groups.google.com/forum/#!forum/salt-users) and
    [salt-announce](https://groups.google.com/forum/#!forum/salt-announce)
    mailing lists. The announcement contains a description of the issue and a
    link to the full release documentation and download locations.

## Receiving security announcements

Keep an eye on the
[Salt Project Security Announcements](https://saltproject.io/security-announcements/)
landing page. Salt Project recommends subscribing to the

> [Salt Project Security RSS feed](https://saltproject.io/feed/?post_type=security)

Reviewer 5: ```suggestion
[Salt Project Security RSS feed](https://saltproject.io/security-announcements/index.xml)
```

to receive notification when new information is available regarding security
announcements.

Other channels to receive security announcements include the

> [Salt Community mailing list](https://groups.google.com/forum/#!forum/salt-users)

Reviewer 5: ```suggestion
[Salt Project GitHub Discussions](https://github.com/saltstack/salt/discussions)
```

and the

> [Salt Project Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg).

Reviewer 5: ```suggestion
[Salt Project Community Discord](https://discord.gg/J7b7EscrAs).
```
